File: ES_NEXT/ESD02/ESD02-DotNet/lecturedemos/Student.js
import { Person } from './Person.js';
export default class Student extends Person {
    constructor(_id, _name, _intake) {
        // this.ID=
        super(_id, _name);
        this.Intake = _intake;
    }
    Print() {
        return `${super.Print()} , Intake = ${this.Intake}`;
    }
}

File: AdvancedC#/lab3/Day3sl/Demo3/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Demo3
{
    internal class Student:Person
    {
        public char Grade { get; set; }
        public override string ToString()
        {
            return $"Student Data- {Id}:{Name}:{Grade}";
        }
    }
}
